index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <script
+      src="https://kit.fontawesome.com/fddcccfd38.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Intuit_form_clone</title>
+  </head>
+
+  <body>
+    <header>
+      <div class="credentials">
+        <p class="info">Already have an Intuit account?</p>
+        <p class="info">
+          <a class="info-link" href="#">I forgot my User ID or Password</a>
+        </p>
+      </div>
+      <div class="sign_in">
+        <button
+          type="submit"
+          class="sign-in-btn common-background parent-position"
+        >
+          <i class="fas fa-unlock-alt child-position"></i>
+          Sign in
+        </button>
+      </div>
+    </header>
+    <div class="main-logo flex-display">
+      <a href="#"
+        ><img class="main-logo-image" src="images/intuit_logo.png" alt="Intuit"
+      /></a>
+    </div>
+    <div class="small-logos flex-display">
+      <div>
+        <a href="#"
+          ><img class="mint-logo" src="images/mint_logo.png" alt="mint"
```

Comment:
* [ ] It shows the Mint logo and text (logo on the left, text on the right) ([link to page](https://accounts.intuit.com/signup.html))